explorer/graphql/data/chains_provider.go

```diff
@@ -15,31 +16,31 @@ type ChainsProvider struct {
 }
 
 // NewChainsProvider creates a new [ChainsProvider] instance with the chains for the given network (e.g. "devnet", "testnet" or "mainnet").
-func NewChainsProvider(network string) *ChainsProvider {
+func NewChainsProvider(network netconf.ID) *ChainsProvider {
 	// Define the chains supported by each network
-	networks := map[string][]struct {
+	networks := map[netconf.ID][]struct {
 		id      uint64
 		name    string
 		logoURL string
 	}{
-		"devnet": {
+		netconf.Devnet: {
 			{id: 1651, name: "Omni Ephemeral", logoURL: "https://chainlist.org/unknown-logo.png"},
 			{id: 1652, name: "Mock L1 Fast", logoURL: "https://chainlist.org/unknown-logo.png"},
 			{id: 1653, name: "Mock L1 Slow", logoURL: "https://chainlist.org/unknown-logo.png"},
 			{id: 1654, name: "Mock L2", logoURL: "https://chainlist.org/unknown-logo.png"},
```

Comment:
Do we need to change this to mock op / mock arb? 
See https://github.com/omni-network/omni/pull/1041

Reply:
I'll change those.